### ISSUE TITLE
Spyder 3.0.1 failed to start, Could not find Qt platform plugin "xcb"

### ISSUE THREAD

**Issue by bcovariant** [spyder-ide/spyder#3683]
## Description of your problem

After a fresh installation of miniconda, and the installation of Spyder, Spyder 3.0.1 fails to start from the console with the error message

> 
> This application failed to start because it could not find or load the Qt platform plugin "xcb"
> in .
> 
> Available platform plugins are: xcb (from /home/beberhardt/miniconda3/pkgs/qt-5.6.0-1/plugins/platforms), minimal, offscreen, xcb.
> 
> Reinstalling the application may fix this problem.
> [1]    9578 abort (core dumped)  spyder
> 

**Please provide any additional information below**

Searching for "xcb" I found two closed issues https://github.com/spyder-ide/spyder/search?q=xcb&state=closed&type=Issues&utf8=%E2%9C%93

It was suggested to start Spyder by specifying the path

> QT_API=pyqt5 QT_QPA_PLATFORM_PLUGIN_PATH=${CONDA_ENV_PATH}/lib/qt5/plugins/platforms spyder

However, this does not work.

## Versions and main components

* Spyder Version: 3.0.1
* Python Version: Python 3.5.2
* Operating system: Linux manjaro 4.4.31-1-MANJARO
* pyqt                      5.6.0                    py35_0  
* qt                        5.6.0                         1  
* qtawesome                 0.3.3                    py35_0  
* qtconsole                 4.2.1                    py35_1  
* qtpy                      1.1.2                    py35_0  

## Dependencies

Please go to the menu entry `Help > Optional Dependencies` (or
`Help > Dependencies`), press the button `Copy to clipboard`
and paste the contents below:

Inside Spyder? It does not start.

**Comment by ccordoba12** [comment]
Are you using Anaconda or the packages provided by your distribution?

**Comment by bcovariant** [comment]
Everything was installed with miniconda/conda, i.e. without anaconda. Then I installed anaconda, however, the same error occurs. Spyder 2.3.9 does work, only the versions 3.x do no at the moment.

**Comment by vroomanj** [comment]
I am experiencing this same issue.

**Comment by ccordoba12** [comment]
This could be caused by the lack of libselinux in your systems, or because you have `LD_LIBRARY_PATH` set (which messes with Anaconda).

**Comment by vroomanj** [comment]
I somehow resolved this issue but not I'm getting the error in issue: https://github.com/spyder-ide/spyder/issues/3545

```
>spyder 
Traceback (most recent call last):
  File "/usr/local/anaconda/bin/spyder", line 6, in <module>
    sys.exit(spyder.app.start.main())
  File "/usr/local/anaconda/lib/python2.7/site-packages/spyder/app/start.py", line 103, in main
    from spyder.app import mainwindow
  File "/usr/local/anaconda/lib/python2.7/site-packages/spyder/app/mainwindow.py", line 49, in <module>
    requirements.check_qt()
  File "/usr/local/anaconda/lib/python2.7/site-packages/spyder/requirements.py", line 39, in check_qt
    import qtpy
  File "/usr/local/anaconda/lib/python2.7/site-packages/qtpy/__init__.py", line 125, in <module>
    raise PythonQtError('No Qt bindings could be found')
qtpy.PythonQtError: No Qt bindings could be found
```
I don't have an LD_LIBRARY_PATH set and SELinux is set to Permissive.

**Comment by bcovariant** [comment]
I have solved the issue by pip uninstalling PyQt5/QtPy, then reinstalling. There seems to be some conflict between the pip/Arch linux/ conda installed libraries that causes this issue (I could also install these packages via the system package manager but using pip seems to work better in this case).


libselinux was installed on my system.

**Comment by ccordoba12** [comment]
@vroomanj, what happens if you uninstall Anaconda and install it in your home directory?

**Comment by DougWellington** [comment]
In my case, I installed Anaconda 4.2.0 as the root user, but then only root could start Spyder.  Everybody else got the error about xcb.  After troubleshooting, I found that a lot of files were installed without world read or execute permission.  Correcting the permissions solved the problem and other users could work with Spyder.

Relevant commands:
find . ! -perm /o=r -ls
find . ! -perm /o=r -exec chmod o+rx {} \;
